design-docs/README.md

```diff
@@ -0,0 +1,23 @@
+# Design Docs Guidelines
+
+In this folder are design documents that describe new feature additions to WPILib. These documents are submitted before significant work on a new feature is started, so that the community has time to examine, criticize, and improve on features before they are added to WPILib. Please follow the guidelines outlined here when submitting a design-doc pull request.
+
+## Format
+
+Design docs should be a markdown-formated text file, with a `.md` extension for proper rendering on GitHub. Documents should have a short, descriptive name, such as `hal_resouce_management.md`, with underlines instead of spaces in the name. All design documents should be added to the master list in this README, with a short 1-2 sentence description. There is no one standard format for the design document itself, but documents should answer the following questions:

```

Comment:
I'm not a fan of having docs as either or. We should be consistent on this. I prefer Markdown, as that's what I know.